tests/conftest.py

```diff
@@ -288,3 +288,9 @@ def add_message_to_group_request_data(study_group0):
         "groupId": study_group0.study_group_id,
         "message": "Test",
     }
+
+
+@pytest.fixture
+def authorized_second_client(client, persist_second_user):
```

Comment:
What's wrong with the first client?

Reply:
Since the persist_review uses persist_second_user, the logged-in user must be persist_second_user.